### PR TITLE
v0.8.0 – Package updates and changelog added

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- CHANGELOG (changelog inception ✨)

### Changed
- Updated dependencies for `gulp-build-fozzie` and `fozzie-colour-palette`
- `hr` styling updated to use `border` rather than `background` as more native

### Removed
- `@debug` statement in `line-height` function

### CODE REVIEW

File: src/scss/base/_typography.scss [hunk 1]
```diff
@@ -222,8 +222,8 @@ samp { font-family: $font-family-mono; }
 hr {
     margin: ($baseline) 0;
     border: 0;
-    height: 1px;
-    background-color: $hr-color;
+    border-top: 1px solid $hr-color;
+    border-bottom: 1px solid #fff;
```
Comment (DamianMullins) [inline src/scss/base/_typography.scss]: Is it OK to use the hex value directly here?

**Comment by DamianMullins** [comment]
Nice work on the changelog, must have taken a while to put together!

File: src/scss/base/_typography.scss (hunk 1 above)
Comment (ashleynolan) [inline src/scss/base/_typography.scss]: Nope – have changed

**Comment by ashleynolan** [comment]
@DamianMullins It’ll learn me to add them from the start of a module from now on 😉

**Pull request merged**